[2021-05-08 01:24:20,143] {scheduler_job.py:182} INFO - Started process (PID=457668) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:24:20,145] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:24:20,146] {logging_mixin.py:104} INFO - [2021-05-08 01:24:20,146] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:24:20,185] {logging_mixin.py:104} INFO - [2021-05-08 01:24:20,176] {dagbag.py:308} ERROR - Failed to import: /home/navneetsajwan/airflow/dags/dag_3.py
Traceback (most recent call last):
  File "/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/navneetsajwan/airflow/dags/dag_3.py", line 26, in <module>
    from  mysql_to_csv import tap_mysql_target_csv,today_dir,move_files,detect_schemas,state_file
ModuleNotFoundError: No module named 'mysql_to_csv'
[2021-05-08 01:24:20,185] {scheduler_job.py:641} WARNING - No viable dags retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:24:20,356] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.228 seconds
[2021-05-08 01:25:17,500] {scheduler_job.py:182} INFO - Started process (PID=458642) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:25:17,524] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:25:17,533] {logging_mixin.py:104} INFO - [2021-05-08 01:25:17,533] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:25:17,581] {logging_mixin.py:104} INFO - [2021-05-08 01:25:17,580] {dagbag.py:308} ERROR - Failed to import: /home/navneetsajwan/airflow/dags/dag_3.py
Traceback (most recent call last):
  File "/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/navneetsajwan/airflow/dags/dag_3.py", line 26, in <module>
    from  mysql_to_csv import tap_mysql_target_csv,today_dir,move_files,detect_schemas,state_file
ModuleNotFoundError: No module named 'mysql_to_csv'
[2021-05-08 01:25:17,582] {scheduler_job.py:641} WARNING - No viable dags retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:25:18,171] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.699 seconds
[2021-05-08 01:26:27,739] {scheduler_job.py:182} INFO - Started process (PID=459890) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:26:27,743] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:26:27,748] {logging_mixin.py:104} INFO - [2021-05-08 01:26:27,744] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:26:27,785] {logging_mixin.py:104} INFO - [2021-05-08 01:26:27,778] {dagbag.py:308} ERROR - Failed to import: /home/navneetsajwan/airflow/dags/dag_3.py
Traceback (most recent call last):
  File "/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/navneetsajwan/airflow/dags/dag_3.py", line 26, in <module>
    from  mysql_to_csv import tap_mysql_target_csv,today_dir,move_files,detect_schemas,state_file
ModuleNotFoundError: No module named 'mysql_to_csv'
[2021-05-08 01:26:27,787] {scheduler_job.py:641} WARNING - No viable dags retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:26:27,962] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.254 seconds
[2021-05-08 01:27:14,454] {scheduler_job.py:182} INFO - Started process (PID=460698) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:27:14,463] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:27:14,466] {logging_mixin.py:104} INFO - [2021-05-08 01:27:14,466] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:27:14,490] {logging_mixin.py:104} INFO - [2021-05-08 01:27:14,481] {dagbag.py:308} ERROR - Failed to import: /home/navneetsajwan/airflow/dags/dag_3.py
Traceback (most recent call last):
  File "/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/navneetsajwan/airflow/dags/dag_3.py", line 26, in <module>
    from  mysql_to_csv import tap_mysql_target_csv,today_dir,move_files,detect_schemas,state_file
ModuleNotFoundError: No module named 'mysql_to_csv'
[2021-05-08 01:27:14,492] {scheduler_job.py:641} WARNING - No viable dags retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:27:14,631] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.183 seconds
[2021-05-08 01:28:08,790] {scheduler_job.py:182} INFO - Started process (PID=461794) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:28:08,796] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:28:08,801] {logging_mixin.py:104} INFO - [2021-05-08 01:28:08,801] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:28:08,842] {logging_mixin.py:104} INFO - [2021-05-08 01:28:08,840] {dagbag.py:308} ERROR - Failed to import: /home/navneetsajwan/airflow/dags/dag_3.py
Traceback (most recent call last):
  File "/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/navneetsajwan/airflow/dags/dag_3.py", line 26, in <module>
    from  mysql_to_csv import tap_mysql_target_csv,today_dir,move_files,detect_schemas,state_file
ModuleNotFoundError: No module named 'mysql_to_csv'
[2021-05-08 01:28:08,845] {scheduler_job.py:641} WARNING - No viable dags retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:28:09,023] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.240 seconds
[2021-05-08 01:29:30,326] {scheduler_job.py:182} INFO - Started process (PID=463027) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:29:30,328] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:29:30,330] {logging_mixin.py:104} INFO - [2021-05-08 01:29:30,330] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:29:30,342] {logging_mixin.py:104} INFO - [2021-05-08 01:29:30,341] {dagbag.py:308} ERROR - Failed to import: /home/navneetsajwan/airflow/dags/dag_3.py
Traceback (most recent call last):
  File "/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/navneetsajwan/airflow/dags/dag_3.py", line 26, in <module>
    from  mysql_to_csv import tap_mysql_target_csv,today_dir,move_files,detect_schemas,state_file
ModuleNotFoundError: No module named 'mysql_to_csv'
[2021-05-08 01:29:30,343] {scheduler_job.py:641} WARNING - No viable dags retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:29:31,928] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 1.607 seconds
[2021-05-08 01:30:19,392] {scheduler_job.py:182} INFO - Started process (PID=463993) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:30:19,394] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:30:19,395] {logging_mixin.py:104} INFO - [2021-05-08 01:30:19,395] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:30:19,411] {logging_mixin.py:104} INFO - [2021-05-08 01:30:19,410] {dagbag.py:308} ERROR - Failed to import: /home/navneetsajwan/airflow/dags/dag_3.py
Traceback (most recent call last):
  File "/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/navneetsajwan/airflow/dags/dag_3.py", line 26, in <module>
    from  mysql_to_csv import tap_mysql_target_csv,today_dir,move_files,detect_schemas,state_file
ModuleNotFoundError: No module named 'mysql_to_csv'
[2021-05-08 01:30:19,412] {scheduler_job.py:641} WARNING - No viable dags retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:30:19,519] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.130 seconds
[2021-05-08 01:31:10,179] {scheduler_job.py:182} INFO - Started process (PID=464706) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:31:10,180] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:31:10,181] {logging_mixin.py:104} INFO - [2021-05-08 01:31:10,181] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:31:10,193] {logging_mixin.py:104} INFO - [2021-05-08 01:31:10,192] {dagbag.py:308} ERROR - Failed to import: /home/navneetsajwan/airflow/dags/dag_3.py
Traceback (most recent call last):
  File "/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/navneetsajwan/airflow/dags/dag_3.py", line 26, in <module>
    from  mysql_to_csv import tap_mysql_target_csv,today_dir,move_files,detect_schemas,state_file
ModuleNotFoundError: No module named 'mysql_to_csv'
[2021-05-08 01:31:10,193] {scheduler_job.py:641} WARNING - No viable dags retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:31:10,337] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.161 seconds
[2021-05-08 01:31:59,199] {scheduler_job.py:182} INFO - Started process (PID=465786) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:31:59,200] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:31:59,201] {logging_mixin.py:104} INFO - [2021-05-08 01:31:59,201] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:31:59,209] {logging_mixin.py:104} INFO - [2021-05-08 01:31:59,209] {dagbag.py:308} ERROR - Failed to import: /home/navneetsajwan/airflow/dags/dag_3.py
Traceback (most recent call last):
  File "/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/navneetsajwan/airflow/dags/dag_3.py", line 26, in <module>
    from  mysql_to_csv import tap_mysql_target_csv,today_dir,move_files,detect_schemas,state_file
ModuleNotFoundError: No module named 'mysql_to_csv'
[2021-05-08 01:31:59,210] {scheduler_job.py:641} WARNING - No viable dags retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:31:59,369] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.174 seconds
[2021-05-08 01:32:57,806] {scheduler_job.py:182} INFO - Started process (PID=466598) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:32:57,809] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:32:57,810] {logging_mixin.py:104} INFO - [2021-05-08 01:32:57,810] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:32:59,706] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_3']) retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:32:59,776] {logging_mixin.py:104} INFO - [2021-05-08 01:32:59,775] {dag.py:1824} INFO - Sync 1 DAGs
[2021-05-08 01:32:59,815] {logging_mixin.py:104} INFO - [2021-05-08 01:32:59,815] {dag.py:1843} INFO - Creating ORM DAG for dag_3
[2021-05-08 01:32:59,861] {logging_mixin.py:104} INFO - [2021-05-08 01:32:59,861] {dag.py:2280} INFO - Setting next_dagrun for dag_3 to 2021-05-06T00:00:00+00:00
[2021-05-08 01:33:00,207] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 2.424 seconds
[2021-05-08 01:33:55,254] {scheduler_job.py:182} INFO - Started process (PID=467759) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:33:55,256] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:33:55,257] {logging_mixin.py:104} INFO - [2021-05-08 01:33:55,257] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:33:56,075] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_3']) retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:33:56,090] {logging_mixin.py:104} INFO - [2021-05-08 01:33:56,089] {dag.py:1824} INFO - Sync 1 DAGs
[2021-05-08 01:33:56,107] {logging_mixin.py:104} INFO - [2021-05-08 01:33:56,107] {dag.py:2280} INFO - Setting next_dagrun for dag_3 to 2021-05-06T00:00:00+00:00
[2021-05-08 01:33:56,225] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.979 seconds
[2021-05-08 01:34:45,044] {scheduler_job.py:182} INFO - Started process (PID=468470) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:34:45,045] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:34:45,046] {logging_mixin.py:104} INFO - [2021-05-08 01:34:45,046] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:34:45,658] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_3']) retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:34:45,673] {logging_mixin.py:104} INFO - [2021-05-08 01:34:45,672] {dag.py:1824} INFO - Sync 1 DAGs
[2021-05-08 01:34:45,690] {logging_mixin.py:104} INFO - [2021-05-08 01:34:45,690] {dag.py:2280} INFO - Setting next_dagrun for dag_3 to 2021-05-06T00:00:00+00:00
[2021-05-08 01:34:45,804] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.763 seconds
[2021-05-08 01:35:39,945] {scheduler_job.py:182} INFO - Started process (PID=469601) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:35:39,963] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:35:39,974] {logging_mixin.py:104} INFO - [2021-05-08 01:35:39,970] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:35:40,897] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_3']) retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:35:40,918] {logging_mixin.py:104} INFO - [2021-05-08 01:35:40,918] {dag.py:1824} INFO - Sync 1 DAGs
[2021-05-08 01:35:40,944] {logging_mixin.py:104} INFO - [2021-05-08 01:35:40,944] {dag.py:2280} INFO - Setting next_dagrun for dag_3 to 2021-05-06T00:00:00+00:00
[2021-05-08 01:35:41,385] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 1.452 seconds
[2021-05-08 01:36:32,560] {scheduler_job.py:182} INFO - Started process (PID=470349) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:36:32,564] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:36:32,565] {logging_mixin.py:104} INFO - [2021-05-08 01:36:32,565] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:36:33,169] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_3']) retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:36:33,188] {logging_mixin.py:104} INFO - [2021-05-08 01:36:33,188] {dag.py:1824} INFO - Sync 1 DAGs
[2021-05-08 01:36:33,209] {logging_mixin.py:104} INFO - [2021-05-08 01:36:33,209] {dag.py:2280} INFO - Setting next_dagrun for dag_3 to 2021-05-06T00:00:00+00:00
[2021-05-08 01:36:33,321] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.765 seconds
[2021-05-08 01:37:20,820] {scheduler_job.py:182} INFO - Started process (PID=471366) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:37:20,821] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:37:20,822] {logging_mixin.py:104} INFO - [2021-05-08 01:37:20,822] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:37:21,292] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_3']) retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:37:21,306] {logging_mixin.py:104} INFO - [2021-05-08 01:37:21,306] {dag.py:1824} INFO - Sync 1 DAGs
[2021-05-08 01:37:21,326] {logging_mixin.py:104} INFO - [2021-05-08 01:37:21,325] {dag.py:2280} INFO - Setting next_dagrun for dag_3 to 2021-05-06T00:00:00+00:00
[2021-05-08 01:37:21,434] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 0.617 seconds
[2021-05-08 01:38:05,869] {scheduler_job.py:182} INFO - Started process (PID=472019) to work on /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:38:05,871] {scheduler_job.py:629} INFO - Processing file /home/navneetsajwan/airflow/dags/dag_3.py for tasks to queue
[2021-05-08 01:38:05,872] {logging_mixin.py:104} INFO - [2021-05-08 01:38:05,872] {dagbag.py:451} INFO - Filling up the DagBag from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:38:07,028] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_3']) retrieved from /home/navneetsajwan/airflow/dags/dag_3.py
[2021-05-08 01:38:07,057] {logging_mixin.py:104} INFO - [2021-05-08 01:38:07,057] {dag.py:1824} INFO - Sync 1 DAGs
[2021-05-08 01:38:07,169] {logging_mixin.py:104} INFO - [2021-05-08 01:38:07,168] {dag.py:2280} INFO - Setting next_dagrun for dag_3 to 2021-05-06T00:00:00+00:00
[2021-05-08 01:38:07,331] {scheduler_job.py:190} INFO - Processing /home/navneetsajwan/airflow/dags/dag_3.py took 1.481 seconds
